.github/config.yml

```diff
@@ -20,6 +20,6 @@ firstPRMergeComment: >
 
   <a href="https://www.jenkins.io/participate/" target="_blank">
    <picture>
-     <img width="600" src="https://raw.githubusercontent.com/jenkinsci/jenkins/master/.github/images/jenkins-welcome.svg">
+     <img width="600" alt=${%Jenkins welcome page} src="https://raw.githubusercontent.com/jenkinsci/jenkins/master/.github/images/jenkins-welcome.svg">
```

Comment:
This applied the wrong suggestion from the comment thread.

Reply:
(Also I'm not sure this particular alt text is good, but that's a separate problem. I encourage you to consider what the image represents on this page, and how that would be accomplished in `alt` text.)

Reply:
Resolved in 87d4ab46b2b736176cc328e53e70bf6169663774